[7051] - Some fields should be read-only (#4245)
DFE-Digital/register-trainee-teachers: app/views/api_docs/reference/v0.1/_reference.md

      </p>
    </dd>
  </div>

>   <div class="govuk-summary-list__row govuk-summary-list__row--no-actions">

Author: 👍

        string
      </p>
      <p class="govuk-body">

>         The ethnicity of the trainee. Coded according to the <a href="https://www.hesa.ac.uk/collection/c23053/e/ethnic">HESA ethnicity field</a>. The <code>ethnic_background</code> and <code>ethnic_group</code> will be determined by the API based on the <code>ethnicity</code> value.

Reviewer: Could we re-phrase this slightly:
```suggestion
        The ethnicity of the trainee. Coded according to the <a href="https://www.hesa.ac.uk/collection/c23053/e/ethnic">HESA ethnicity field</a>. The values for <code>ethnic_background</code> and <code>ethnic_group</code> will be set based on the <code>ethnicity</code> value.
```
